src/cdk/drag-drop/drag.ts

```diff
@@ -321,13 +321,13 @@ export class CdkDrag<T = any> implements AfterViewInit, OnDestroy {
   }
 
   /** Gets only handles that are not inside descendant `CdkDrag` instances. */
-  private getChildHandles() {
+  protected _getChildHandles() {
```

Comment:
We might be inconsistent about this, but the coding standards doc says no prefix for `protected`

Reply:
Doing a quick code search shows that most of the protected APIs we have at the moment are underscored.

Reply:
Ack, let's revisit this another time